libdrm/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_name=libdrm
+pkg_origin=core
+pkg_version=2.4.88
+pkg_description="Direct Rendering Manager"
+pkg_upstream_url="https://dri.freedesktop.org/wiki/"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_source="https://dri.freedesktop.org/${pkg_name}/${pkg_name}-${pkg_version}.tar.gz"
+pkg_shasum=a8b458db6a73c717baee2e249d39511fb6f5c0f5f24dee2770935eddeda1a017
+pkg_deps=(
+  core/libpciaccess
+  core/glibc
+)
+pkg_build_deps=(
+  core/cairo   # not linked by bins/libs
```

Comment:
Ultimately I think this is for 2 reasons: 

1. On my machine the the configure script is failing to find some of cairo's deps via pkg-config:

        checking for CAIRO... no

    I was able to get it working by adding all of the following:

        core/glib
        core/cairo   # not linked by bins/libs
        core/pcre
        core/pixman
        core/fontconfig
        core/freetype
        core/zlib
        core/libpng
        core/expat

2. As far as I can see from some quick grepping of the source, this is only required to enable some additional tests (and the configure script seems to confirm this) so it won't be linked into the libs.  Given the number of dependencies, I'm not sure it is worth it really.

Thus, I think one option here is to just drop this dep and perhaps leave a comment in this file about why.

Reply:
Or we could add them to build_deps instead? If it tests that it works well with cairo and people actually use a cairo package, might be safer?